backend/routers/services.py

```
@@ -2,67 +2,77 @@
 
 async def get_dynamic_pricing(db, service_name: str = ""):
-    """Get dynamic pricing from admin settings"""
+    """Get dynamic pricing from admin settings with enhanced comprehensive pricing"""
     try:
-        # Get global pricing multiplier
-        pricing_multiplier = await db.fetchrow("""
-            SELECT value FROM platform_settings 
-            WHERE key = 'pricing_multiplier'
-        """)
+        # Try to get enhanced comprehensive pricing first
+        enhanced_pricing = await get_enhanced_comprehensive_pricing(db, service_name)
```

Ruff (0.12.2)

Undefined name `get_enhanced_comprehensive_pricing`

(F821)